CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# Terraform Provider MSO - Changelog
+
+All notable changes to this project will be documented in this file.
+
+## 1.6.0 (November 5, 2025)
+
+IMPROVEMENTS:
+- Add retrigger attribute to mso_rest to enable updates without payload, path or method changes.
+- Add ability to deploy tenant_policies in mso_schema_template_deploy_ndo.
```

Comment:
I think it is the ability to deploy all template types not just tenant_policies by adding template_type and template_id attributes.